src/cljc/witan/send/params.cljc

```diff
@@ -300,16 +303,18 @@
                                {} observations)
         prior-per-year (continue-for-latter-ays prior-per-year academic-years)]
     (reduce (fn [coll [ay state]]
-              (if-let [beta-params (merge-with +
-                                               (get-in observations [ay state])
-                                               (get prior-per-year ay))]
-                (assoc coll [ay state] beta-params)
+              (if (any-valid-transitions? (count (get valid-transitions (second (s/need-setting state)))))
```

Comment:
I was thinking the call here would look like `(if (any-valid-transition? valid-transitions state)`